spiceaidocs/content/en/reference/Connectors/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+type: docs
+title: "Connectors"
+linkTitle: "Connectors"
+description: ''
+weight: 70
+---
+
+## Connectors
```

Comment:
`Connectors` -> `Data Connectors` in all places.

Reply:
title not needed here, we have it from template
![CleanShot 2024-03-15 at 11 04 49@2x](https://github.com/spiceai/docs/assets/827338/8e157aca-d9b7-4b34-96d0-09dabbaecce4)